revocation/crl/fetcher/fetcher.go

```diff
@@ -0,0 +1,158 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package fetcher provides Fetcher interface and its implementation to fetch
+// CRL from the given URL
+package fetcher
```

Comment:
What if `fetcher`? Does this package depend on #214?

Reply:
It doesn’t depend on #214. The CRL will be refactored to use Fetcher to get the CRL from the cache or by downloading it. It is also for preloading.